### ISSUE TITLE
Says unprocessible entity, but still posts the data

### ISSUE THREAD

**Issue by iolocWebsite** [remy/jsonbin#41]

**Comment by iolocWebsite** [comment]
![image](https://user-images.githubusercontent.com/55917515/67396078-448dd480-f5a7-11e9-80e4-02a9d32de841.png)

**Comment by richcorbs** [comment]
I'm getting the same thing from CURL and javascript FETCH.

**Comment by oddboy** [comment]
Another one here.  curl and also using the jsonbin Web Service (AutoWeb) for Tasker.

curl --version
curl 7.29.0 (x86_64-redhat-linux-gnu) libcurl/7.29.0 NSS/3.36 zlib/1.2.7 libidn/1.28 libssh2/1.4.3
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smtp smtps telnet tftp
Features: AsynchDNS GSS-Negotiate IDN IPv6 Largefile NTLM NTLM_WB SSL libz unix-sockets

AutoWeb (v 1.0.beta.2) + Tasker 5.8.5

**Comment by virtualdj** [comment]
Same for me, like @oddboy with Tasker 5.9.1 and AutoWeb 1.0.beta2!

**Comment by ekkis** [comment]
is this service no longer supported?

**Comment by remy** [comment]
It was free from the outset so there was never any support. But it's also open source which means that *we* can all make it better of there's a problem we face.

I hope that answers your question.

**Comment by ekkis** [comment]
it does, thank you.  I spent a bit of time with the source trying to figure out where the issue emanates but I'm limited on the time I can devote to it.  the problem occurs on a base usecase and needs some attention.  for now I'm just ignoring the return value and reissuing a query to make sure the value got written down but that's not a good thing to do.  I've opened a separate issue on this

**Comment by remy** [comment]
I may have found the root cause of this - mongoose doesn't might be trying to write in parallel on the same record. I've added a promise to handle this potential situation and will close this issue for the time being (in the vague hope it's fixed :))
